internal/backuprestore/backup.go

```diff
@@ -55,7 +57,7 @@ func (h *BRHandler) CheckIfBackupRequested(content []lcav1alpha1.ConfigMapRef, c
 	}
 
 	// extract CM
-	oadpConfigmaps, err := getConfigMaps(ctx, h, content)
+	oadpConfigmaps, err := common.GetConfigMaps(ctx, h, content)
```

Comment:
Not related to your change, but should this be?
```suggestion
	oadpConfigmaps, err := common.GetConfigMaps(ctx, h.client, content)
```

Reply:
It should be indeed! I'll include the change :)

Reply:
Thanks!